demos/background_subtraction_demo/cpp_gapi/background_subtraction_demo_gapi.hpp

```diff
@@ -33,6 +33,11 @@ static const char blur_bgr_message[] = "Optional. Blur background.";
 static const char target_bgr_message[] =
     "Optional. Background onto which to composite the output (by default to green field).";
 static const char utilization_monitors_message[] = "Optional. List of monitors to show initially.";
+static const char use_onevpl_message[] = "Optional. Use onevpl video decoding.";
+static const char onevpl_params_message[] = "Optional. Parameters for onevpl video decoding. "
+                                            "Format: <prop name>:<value>;<prop name>:<value> "
+                                            "Semicolon separated list of oneVPL mfxVariants which is used for configuring source "
+                                            "(see `MFXSetConfigFilterProperty` by https://spec.oneapi.io/versions/latest/elements/oneVPL/source/index.html)";
```

Comment:
+ @sivanov-work 
Could you comment, please?

Reply:
Example: --cfg_params="mfxImplDescription.mfxDecoderDescription.decoder.CodecID:MFX_CODEC_HEVC;"

Reply:
I think there is no single place where all the information would be collected, so that's why here the more generic link.
Agree that it's difficult to understand how to use it, probably the example above will help?

Reply:
Yes, please, add the example. Is https://github.com/opencv/opencv/wiki/Graph-API#vpl-source-capabilities--limitations more suitable link?

Reply:
@Wovchena 
These must be fine-grained VPL instance configuration params and they are passed as walk-through into directly VPL library configuration routine. Because VPL documentations doesn't provide the actual list of these parameters then we consider that user should determine them and apply by yourself (by using oneVPL link).  These parameters set is depend from customer purpose and doens't limited by gapi-vpl implementation (there are actual limitations, but idea is the same)
So if you want to configure VPL instance with some specific params (like implementation version and so on) you should put it here in the form <param_name:param_value> exactly with separator ":". It develoved as a way to allow fine-grained VPL instance customizations

BUT there are several important parameters here, like codec type (mfxImplDescription.mfxDecoderDescription.decoder.CodecID), acceleration type and several of them. These kind of parameters enumerated in 'onevpl/cfg_params.hpp'. These parameters also belong to the VPL parameters list, but now it wrapped in `cfg_params.hpp` just for quick referencing.

Reply:
Finally, I hope that casual parameters (which should be used more often that another) are the following:
-mfxImplDescription.mfxDecoderDescription.decoder.CodecID
the values are https://spec.oneapi.io/onevpl/2.7.0/API_ref/VPL_enums.html?highlight=mfx_codec_hevc#codecformatfourcc
-mfxImplDescription.AccelerationMode
the allowable values are
https://spec.oneapi.io/onevpl/2.7.0/API_ref/VPL_disp_api_enum.html?highlight=d3d11#mfxaccelerationmode

Reply:
Please pay attention that GAPI VPL  component was not tested in user-friendly way because there were no actual customers for it. And any ideas which are called to improve its usability are preferred

Reply:
@TolyaTalamanov, please, adopt @sivanov-work's comments for flag description. That's fine if it will be long.

Reply:
Done